service/src/main/java/com/ge/predix/acs/monitoring/AcsMonitoringUtilities.java

```diff
@@ -62,6 +75,34 @@ static Health health(final Supplier<HealthCode> check, final String description)
         }
     }
 
+    static Health cacheHealth(final RedisConnectionFactory connectionFactory, final String cacheType,
```

Comment:
You can move this method to the `AbstractCacheHealthIndicator` class since it's only used by that class and not across all health indicators.